.vscode/settings.json

```diff
@@ -13,11 +13,11 @@
   "json.schemas": [
     {
       "fileMatch": ["*.docs.json"],
-      "url": "./script/components-json/component.schema.json"
+      "url": "./packages/react/script/components-json/component.schema.json"
     },
     {
       "fileMatch": ["generated/components.json"],
-      "url": "./script/components-json/output.schema.json"
+      "url": "./packages/react/script/components-json/output.schema.json"
```

Comment:
Tiny unrelated change to fix these schema paths so we can get typechecking/documentation for component metadata JSON